tools/sigclient/pkg/utils/metricsreader.go

```diff
@@ -53,15 +71,25 @@ func (mg *MetricsGenerator) GetLogLine() ([]byte, error) {
 }
 
 func (mg *MetricsGenerator) GetRawLog() (map[string]interface{}, error) {
-
 	retVal := make(map[string]interface{})
-	mName := fmt.Sprintf("testmetric%d", fastrand.Uint32n(mg.nMetrics))
+	mName := fmt.Sprintf("testmetric%d", mg.f.Rand.Intn(int(mg.nMetrics)))
 	retVal["metric"] = mName
-	retVal["timestamp"] = time.Now().Unix()
-	if fastrand.Uint32n(1_000)%2 == 0 {
-		mg.val = float64(fastrand.Uint32n(1_000)) - 500 + rand.Float64()
+
+	curTime := time.Now().Unix()
+	if mg.multiTimestampsEnabled {
+		switch mg.f.Rand.Int() % 3 {
+		case 0:
+			retVal["timestamp"] = curTime
+		case 1:
+			retVal["timestamp"] = curTime - 60*60*24*1
+		case 2:
```

Comment:
write a short description of multitimestampenabled means ? and what these values 0,1,2 signify